doc/source/Tutorials/BeginnerTutorial/index.md

```diff
@@ -86,9 +86,9 @@ In the _Viewer_, you can now see the bounding boxes of the objects at several lo
 
 1. In the _Hierarchy View_, <kbd>Shift</kbd> + click the ![triangle](images/collapsibleArrowRight.png "Triangle") next to *C_head_GRP*. All of the children of *C_head_GRP* will expand. Now the _Viewer_ shows all of the geometry that comprises Gaffy's head.
 
-2. <kbd>Shift</kbd> + click the ![triangle](images/collapsibleArrowRight.png "Triangle") next to *L_legUpper_GRP*. All the locations under *L_legUpper_GRP* will expand. Now the _Viewer_ also shows all of the geometry that comprises Gaffy's left leg.
+2. <kbd>Shift</kbd> + click the ![triangle](images/collapsibleArrowRight.png "Triangle") next to *R_legUpper_GRP*. All the locations under *R_legUpper_GRP* will expand. Now the _Viewer_ also shows all of the geometry that comprises Gaffy's right leg.
```

Comment:
Just out of curiosity, what is better about this ordering?

Reply:
The left leg is closer to the default viewport camera position, so for the next section when the user is asked to marquee-select the opposite leg's bounding box, it'll be marginally easier for them to select it instead of the right leg.